### PR TITLE
Add hubspot properties

### PR DESCRIPTION
**Pull request by steinitzu**
# Tell us what you do here

- [x] improving, documenting, customizing existing source (please link an issue or describe below)

# Relevant issue

Implements https://github.com/dlt-hub/verified-sources/issues/195

# More PR info

### CODE REVIEW

**Review by rudolfix — CHANGES_REQUESTED** [review]
looks good! what we still miss:
- show how to fetch history in the demo script
- enable history on the `test_all_reources` (which calls real backend)

**Comment by steinitzu** [comment]
@rudolfix added it to tests and demo. I don't think I have the right test account locally so I'm not checking the data very thoroughly.

File: sources/hubspot/__init__.py [hunk 1]
```diff
@@ -84,17 +88,19 @@ def crm_objects(
...
-    for objects, history_entries in fetch_data_with_history(
-        CRM_OBJECT_ENDPOINTS[object_type], api_key, params=params
-    ):
-        yield objects
-        yield dlt.mark.with_table_name(
-            history_entries, OBJECT_TYPE_PLURAL[object_type] + "_property_history"
-        )
+        # Get history separately, as requesting both all properties and history together
+        # is likely to hit hubspot's URL length limit
+        for history_entries in fetch_property_history(
```
Comment (steinitzu) [inline sources/hubspot/__init__.py:96]: Changed this too. With some endpoints the URL was too long when listing all properties twice. :fearful:

**Comment by rudolfix** [comment]
@steinitzu I sent you the credentials to test account. let's make sure that some proper test cases are there before merging. thanks!

**Review by rudolfix — APPROVED** [review]
OK, LTM!

**Pull request merged**